src/styles/styles.scss

```diff
@@ -0,0 +1,141 @@
+body {
+  background-color: #f5f6fa;
+}
+
+.app {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 20px;
+
+  .main {
+    .filter {
+      border-radius: 4px;
+      border: solid 1px #e8e8e8;
+      background-color: #ffffff;
+      margin-right: 15px;
+      width: 285px;
+      height: 45px;
+      display: block;
+      margin: 0 auto;
+      text-align: center;
+      margin-bottom: 20px;
+    }
+  }
+}
+
+
+.formContainer {
+  border-top: solid 1px #e8e8e8;
+  padding-top: 40px;
+
+  h2 {
+    width: 174px;
+    height: 24px;
+    font-family: Montserrat;
+    font-size: 20px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: -0.18px;
+    color: #888888;
+    text-transform: uppercase;
+  }
+
+  .formBook {
+    display: flex;
+
+    input,
+    select {
+      border-radius: 4px;
+      border: solid 1px #e8e8e8;
+      background-color: #ffffff;
+      margin-right: 15px;
+    }
+  
+    input {
+      width: 664px;
+      height: 45px;
+    }
+  
+    select {
+      width: 285px;
+      height: 45px;
+    }
+  
+    button {
+      width: 184px;
+      height: 45px;
+      border-radius: 3px;
+      background-color: #0290ff;
+      color: #fff;
+      border: none;
+    }
+  }
+ 
+}
+
+table {
+  width: 1200px;
+
+  thead {
+    display: none;
+  }
+
+  tbody {
+    display: flex;
+    flex-direction: column;
+    
+    tr {
+      border-radius: 4px;
+      border: solid 1px #e8e8e8;
+      background-color: #ffffff;
+      display: inline;
+      margin-bottom: 20px;
+      padding: 20px;
+      height: 170px;
+      box-sizing: border-box;
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+
+      th:first-child {
+        display: none;
+      }
+
+      .title {
+        font-family: RobotoSlab;
+        font-size: 22px;
+        font-weight: bold;
+        color: #121212;
+        text-transform: capitalize;
+      }
+
+      .category {
+        opacity: 0.5;
+        font-family: Montserrat;
+        font-size: 14px;
+      }
+
+      .button {
+        margin-top: auto;
+
+        button {
+          width: 54px;
+          height: 19px;
+          font-family: RobotoSlab;
+          font-size: 14px;
+          font-weight: 300;
+          font-stretch: normal;
+          font-style: normal;
+          line-height: normal;
+          letter-spacing: normal;
+          color: #4386bf;
+          border: none;
+          background: none;
+          outline: none;
```

Comment:
You should add `cursor: pointer` so that user can easily see when the button is clicked